src/shared_modules/dbsync/src/sqlite/sqlite_wrapper.cpp

```diff
@@ -232,7 +232,10 @@ void Statement::bind(const int32_t index, const double_t value)
 
 std::string Statement::expand()
 {
-    return sqlite3_sql(m_stmt.get());
+    auto pExpandedSQL{ sqlite3_expanded_sql(m_stmt.get()) };
+    std::string expandedSQLString{ pExpandedSQL };
+    sqlite3_free(pExpandedSQL);
+    return expandedSQLString;
```

Comment:
Add the following code to be compliant with the RAII technique.

on the top of this file the next code:

```
#include "customDeleter.hpp"
using ExpandedSQLPtr = std::unique_ptr<char, CustomDeleter<decltype(&sqlite3_free), sqlite3_free>>;

```

```suggestion
    return ExpandedSQLPtr(sqlite3_expanded_sql(m_stmt.get())).get();
```